addons/web_editor/static/src/js/wysiwyg/widgets/link_tools.js

```diff
@@ -501,7 +501,7 @@ const LinkTools = Link.extend({
             return;
         }
         const protocolLessPrevUrl = previousUrl.replace(/^https?:\/\/|^mailto:/i, '');
-        const content = this._link.innerText;
+        const content = this._link.innerText.trim().replaceAll('\u200B', '');
```

Comment:
> the tour systematically failing because of this?

@qsm-odoo On my laptop at least, it fails every time without this change.
But indeed, there may be something else...
- June 8, MOC saw the error happening (not often) as he said
- Then I suppose he deactivated the test on the runbot (when exactly IDK?)
- September 13, [this PR](https://github.com/odoo/odoo/pull/118676) has been r+, it added the "always failing step" but it's not failing at this date.
- October 5, [this other PR](https://github.com/odoo/odoo/pull/108358) has been r+, which break the step added before.

So I suppose the PR can be merged when the test is deactivated ?

This PR fixes the always failing issue, but maybe the initial issue is still there IDK.
I will ask a multi build to check :+1:

Reply:
Thanks for the explanation. Indeed, ask for a multi build and if it's ok, we'll merge this as it is :+1: